hokusai/commands/env.py

```diff
@@ -16,14 +16,10 @@ def delete_env(context, namespace=None):
 def get_env(context, environment, namespace=None):
   configmap = ConfigMap(context, namespace=namespace)
   configmap.load()
-  if environment:
-    for k, v in configmap.all():
-      if k in environment:
-        print_smart("%s=%s" % (k, v))
-  else:
-    for k, v in configmap.all():
-      print_smart("%s=%s" % (k, v))
-
+  all_configs = configmap.all()
+  for k in sorted(all_configs.keys()):
```

Comment:
More concise (`sorted` will by default to sort by the first element in a tuple):
```
for k, v in sorted(configmap.all().items()):
  if not environment or (environment and k in environment):
    print_smart("%s=%s" % (k, v))
```